.github/workflows/release-integration.yml

```diff
@@ -81,9 +81,17 @@ jobs:
         env:
           GENERATE_PACKAGES: true
           UPLOAD_PACKAGES: ${{ github.event.release.prerelease }}
+      - name: release fips
+        # FIPS packages need to be built in the fips container
+        # TODO: It should be refactored when we change the way docker images are being built
+        run: make ci/prerelease-fips
+        env:
+          GENERATE_PACKAGES: true
+          UPLOAD_PACKAGES: ${{ github.event.release.prerelease }}
+          GPG_MAIL: 'infrastructure-eng@newrelic.com'
+          # The upload script expects this to authenticate.
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
it seems a different approach than normal release. Why in why case is called "make release" and now "make ci/prerelease-fips"?
We have in one "release" and in the other "prerelease" 

Moreover, why for fibs we need as well GPG_MAIL and GITHUB_TOKEN?

Reply:
why is it not a step in "make release"?

Reply:
Yes, it is different. As stated in the PR description, we need to build FIPS packages in docker. To not impact anything unless we plan to remove the image building from `goreleaser`, FIPS is released as a separate step.

> Moreover, why for fibs we need as well GPG_MAIL and GITHUB_TOKEN?

I think it was copy-paste, it might not be needed. I'll remove these.

> why is it not a step in "make release"?

The current flow was cleaning `dist` after each release and upload. To minimize the changes needed, it was implemented this way.